src/main/kotlin/racingcar/view/PromptService.kt

```diff
@@ -1,7 +1,9 @@
 package racingcar.view
 
+import racingcar.domain.car.CarName
+
 interface PromptService {
-    fun getNameOfCars(assignedNames: String? = null): List<String>
+    fun getNameOfCars(assignedNames: String? = null): Set<CarName>
     fun getNumberOfMoves(assignedNumber: Int? = null): Int
-    fun showResult(result: List<Map<String, Int>>)
+    fun showResult(result: List<Map<String, Int>>, winners: Set<CarName>)
```

Comment:
```suggestion
    fun showResult(result: List<CarRacingResult>, winners: Set<CarName>)
```
여기서도 CarRacingResult type을 사용해볼까요?